shipit.polaris-react.yml

```diff
@@ -16,4 +16,5 @@ dependencies:
 
 deploy:
   override:
+    - yarn turbo run build --filter=@shopify/polaris
```

Comment:
I'm assuming turbo.json has been taught that tokens needs to be built before polaris

Reply:
Yes, that is correct! Which makes this much better than just using the `yarn workspace [package] [script]` command since that doesn't execute builds for topological dependencies